examples/user/ns-per-user.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: users.ctl.enisoc.com
+spec:
+  group: ctl.enisoc.com
```

Comment:
Do you have a personal domain you can use here since you wrote the example? If you don't want to use your own domain, let's just use `example.com`.

Reply:
I do, I'll update it to use `rlg.io`, thanks!